tern/formats/default/generator.py

```diff
@@ -34,19 +34,35 @@ def print_full_report(image):
                                                       '\t', '\t\t')
             layer_pkg_list = []
             layer_license_list = []
+            layer_file_licenses_list = []
+            file_level_licenses = None
+
+            for f in layer.files:
+                layer_file_licenses_list.extend(f.license_expressions)
+            layer_file_licenses_list = list(set(layer_file_licenses_list))
+
+            if layer_file_licenses_list:
+                file_level_licenses = ", ".join(layer_file_licenses_list)
+
             for package in layer.packages:
                 pkg = package.name + "-" + package.version
                 if pkg not in layer_pkg_list and pkg:
                     layer_pkg_list.append(pkg)
                 if package.pkg_license not in layer_license_list and \
                         package.pkg_license:
                     layer_license_list.append(package.pkg_license)
-            # Collect packages + licenses in the layer
-            notes = notes + formats.layer_packages_list.format(
-                list=", ".join(layer_pkg_list) if layer_pkg_list else 'None')
-            notes = notes + formats.layer_licenses_list.format(list=", ".join(
-                layer_license_list) if layer_license_list else 'None')
-            notes = notes + formats.package_demarkation
```

Comment:
Actually, we do want to report that no packages and licenses were found:
```
------------------------------------------------

        Packages found in Layer:  None
        Licenses found in Layer:  None
------------------------------------------------
```
The problem is that the layer sha is not reported.
This patch seems to report it:
```
        Layer: d2e6d9a0a0: 
```
But nothing about package and license. We want both:
```
        Layer: d2e6d9a0a0: 
        Packages found in Layer:  None
        Licenses found in Layer:  None
```